### ISSUE TITLE
The line "usingnamespace std.os.windows;" causes zls to terminate until that file is no longer in open tabs + is manually restarted

### ISSUE THREAD

**Issue by GavinRay97** [zigtools/zls#280]
![AjS90gSky6](https://user-images.githubusercontent.com/26604994/113362512-8d10fc00-931c-11eb-9aec-d5c2222b6799.gif)

In case the output/stack trace is useful, I have logging turned on + here it is:
```
```log
Enabled debug logging[Info  - 6:59:43 PM] [info-main] Using zig executable C:\Zig\zig-windows-x86_64-0.8.0-dev.1574+e088a17f5\zig.exe
[Info  - 6:59:43 PM] [info-main] ClientCapabilities{ .supports_snippets = true, .supports_semantic_tokens = true, .hover_supports_md = true, .completion_doc_supports_md = true }
[debug-main] Took 0ms to process method initialize
[debug-main] Took 0ms to process method initialized
[debug-doc_store] Opened document: file:///c%3A/Users/rayga/Projects/tmp/reaper-c/test1.zig
[debug-main] Took 1ms to process method textDocument/didOpen
[debug-doc_store] Opened document: file:///c%3A/Users/rayga/Projects/tmp/reaper-c/main.zig
[debug-main] Took 0ms to process method textDocument/didOpen
[debug-main] Took 0ms to process method textDocument/documentSymbol
[debug-main] Took 0ms to process method textDocument/semanticTokens/full
[debug-main] Took 0ms to process method textDocument/semanticTokens/full
[debug-doc_store] New text for document file:///c%3A/Users/rayga/Projects/tmp/reaper-c/main.zig
[debug-main] Took 0ms to process method textDocument/didChange
[Info  - 6:59:45 PM] Connection to server got closed. Server will restart.
[Error - 6:59:45 PM] Request textDocument/semanticTokens/full failed.
Error: Connection got disposed.
	at Object.dispose (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\connection.js:1044:27)
	at Object.dispose (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-languageclient\lib\common\client.js:73:35)
	at LanguageClient.handleConnectionClosed (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-languageclient\lib\common\client.js:2352:42)
	at LanguageClient.handleConnectionClosed (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-languageclient\lib\node\main.js:155:15)
	at closeHandler (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-languageclient\lib\common\client.js:2339:18)
	at CallbackList.invoke (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\events.js:55:39)
	at Emitter.fire (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\events.js:117:36)
	at closeHandler (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\connection.js:256:26)
	at CallbackList.invoke (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\events.js:55:39)
	at Emitter.fire (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\events.js:117:36)
	at StreamMessageReader.fireClose (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\messageReader.js:40:27)
	at Socket.<anonymous> (c:\Users\rayga\.vscode-insiders\extensions\augusterame.zls-vscode-1.0.4\node_modules\vscode-jsonrpc\lib\common\messageReader.js:125:42)
	at Socket.emit (events.js:327:22)
	at Pipe.<anonymous> (net.js:674:12)
```

**Comment by InterplanetaryEngineer** [comment]
I can not reproduce this, could you give a few more details?

**Comment by alexnask** [comment]
Fixed in #293
